src/vue/main-menu.vue

```diff
@@ -1,32 +1,36 @@
 <template>
   <div class="alpheios-alignment-app-menu" id="alpheios-main-menu">
       <div class="alpheios-alignment-app-menu__buttons">
-        <button class="alpheios-button-tertiary alpheios-menu-button" id ="alpheios-main-menu-add-target" 
-                @click="$emit('add-target')" :disabled="!docSourceEditAvailable" >
+        <button class="alpheios-editor-button-tertiary alpheios-menu-button" id ="alpheios-main-menu-options" 

```

Comment:
I think we may need to add some grouping or additional levels to the main menu now that there are so many buttons. there isn't much logical order to them. 

Options, Upload and Download should be differentiated from actions that one does to create or edit alignments. 

Undo, Redo and Add Target are actions that can be done while aligning, and should be grouped together

Align should be separated from the rest, and more clearly associate with the act of submitting new texts (while creating an alignment from scratch or adding a new one)

We might also want to consider standard icons for Options (Settings), Upload, Download, Undo and Redo

Maybe we need a new issue for all of this though.

Maybe Options, Upload and Download could be smaller icons (e.g. setting wheel for options, standard upload/download icons for those) these probably

Reply:
I think - it is better to create a separate issue for that - https://github.com/alpheios-project/alignment-editor-new/issues/86